lib/inky.ex

```
@@ -185,7 +227,13 @@ defmodule Inky do
 
   @impl GenServer
   def handle_cast(:push, state) do
-    push(:await, state)
+    state =
+      case push(:await, state) do
+        :ok -> state
+        {:error, _} -> state
+        {:ok, state} -> state
+      end
+
```

More silent failures?

---

This change is no longer needed either... reverting.